### DISCUSSION TITLE
Any plans to support AMD GPUs for object detection in 0.14.x?

### DISCUSSION THREAD

**Discussion by ross-rosario** [blakeblackshear/frigate#13162] [Detector Support]
### What is your question:

The AMD card support is what brought me to Frigate 0.14-dev, and I'd like to know whether re-introducing the support (with whatever model, even a stupid one) is planned for the near/mid future, or is completely out of scope.

Thanks.

**Comment by NickM-27** [comment]
I have a PR for adding it, it would be 0.15 at the earliest

**Reply by ross-rosario** [reply]
How can one help with that endeavor? Can a work-in-progress PR be opened so whoever is interested can test it or chime in with an idea?

**Reply by NickM-27** [reply]
there is already a branch, for now there's no default model that can be used so the onnx model needs to be generated via the collab notebook https://github.com/blakeblackshear/frigate/tree/rocm-yolonas

**Reply by ross-rosario** [reply]
Very good. How does that compare to the rknn detector, which also uses yolo-nas model? In other words, is the work to be done to support AMD cards, akin to this PR https://github.com/blakeblackshear/frigate/pull/11365 ?
